docs/develop/edit_a_module.md

```diff
@@ -8,7 +8,7 @@ If it is a backend change, simply installing the module from the local path
 should be enough. In some corner cases, you might have to re-start the
 instance (for that run `stop` and `run` again). For frontend (e.g. templates,
 css, etc.) you need to symlink and re-build the assets. This can be a complex
-process when using webpack, but Invenio-CLI got you covered.
+process when using webpack, but Invenio-CLI has got you covered.
```

Comment:
minor: this doesnt sound goot to me, either `got` either `has`. But I'm not native 😅

Reply:
Yeah, it's an informal idiom: https://www.merriam-webster.com/dictionary/have%20%28got%29%20someone%20or%20something%20covered ... I will change to just "has" to make it formal.